### ISSUE TITLE
Update the convert_models/run.sh

### ISSUE THREAD

**Issue by melodyliulab** [containers/ai-lab-recipes#724]
I want to convert the huggingface models to gguf file, but always met errors when following the README.md, then I found the files in llama.cpp repo are updated:

python download_huggingface.py --model $hf_model_url  --> **this should be converted_models/download_huggingface.py**
python llama.cpp/convert.py /opt/app-root/src/converter/converted_models/$hf_model_url -> **there is no convert.py in https://github.com/ggerganov/llama.cpp**
python llama.cpp/convert-hf-to-gguf.py /opt/app-root/src/converter/converted_models/$hf_model_url -> **llama.cpp/convert-hf-to-gguf.py should be  llama.cpp/convert_hf_to_gguf.py now**

Can you fix the problem and let me try again?

**Comment by rhatdan** [comment]
Care to open a PR?

**Comment by melodyliulab** [comment]
> Care to open a PR?
Sure, please review it https://github.com/containers/ai-lab-recipes/pull/741
